src/Workspaces/Core/Portable/Execution/SerializerService_Reference.cs

```diff
@@ -46,7 +46,7 @@ public static Checksum CreateChecksum(AnalyzerReference reference, CancellationT
 
             using var stream = SerializableBytes.CreateWritableStream();
 
-            using (var writer = new ObjectWriter(stream, leaveOpen: true, canKeepObjectsAlive: false, cancellationToken))
+            using (var writer = new ObjectWriter(stream, leaveOpen: true, canKeepObjectsAlive: true, cancellationToken))
```

Comment:
doc

Reply:
The doc for this usage is on the `TakeKeepAliveObjects` call below (which may be updated, but doesn't need to be in two places and of the two calls, the `TakeKeepAliveObjects` is the less obvious one).